lib/transaction/index.js

```diff
@@ -886,19 +886,14 @@ function acceptDistributedTraceHeaders(transportType, headers) {
 
   if (traceparent) {
     logger.trace('Accepting trace context DT payload for transaction %s', this.id)
-
     // assumes header keys already lowercase
     const tracestate = headers[TRACE_CONTEXT_STATE_HEADER]
-
     this.acceptTraceContextPayload(traceparent, tracestate, transport)
-  } else {
+  } else if (NEWRELIC_TRACE_HEADER in headers) {
+    logger.trace('Accepting newrelic DT payload for transaction %s', this.id)
```

Comment:
This change allows empty string newrelic headers through to the payload method for processing, so that `DistributedTrace/AcceptPayload/Ignored/Null` metrics can be processed.